Library/Formula/tokyo-tyrant.rb

```diff
@@ -15,6 +15,13 @@ class TokyoTyrant < Formula
     patch :DATA
   end
 
+  patch do
+    # Patch to fix UNIX socket issue
+    # https://gist.github.com/bow-fujita/a338e35b95605ad63147
+    url 'https://gist.githubusercontent.com/bow-fujita/a338e35b95605ad63147/raw/3bd1bb84ea58f7a71a485fe032299def3a288576/tokyotyrant-fix-unix-socket.patch'
+    sha1 'dd76282b0ea99104372c8da09e564f308d83ffdf'
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!